app/controllers/answers_controller.rb

```diff
@@ -15,8 +15,10 @@ def create
     @answer = Answer.new
     @questions = get_question_list()
     @question = @questions[(params[:question_id]).to_i - 1]["text"]
-    @answers = @questions[(params[:question_id]).to_i - 1]["answers"]
+    @answers = @question["answers"]
```

Comment:
Looking at this now, it seems like this actually wouldn't work. I didn't realize that @question isn't actually a single list member of `@questions`. Instead, it's the `text` attribute of a Question object. There is no `answers` attribute to the `text` attribute, so line 18 will fail now. Sorry for the confusion, I should've caught that in the first code review.

So two things:
1. Do we actually every use the answers_controller.rb? I can't see where it's used anymore. It appears that we don't have an endpoint that actually calls this create controller. So if you can thoroughly test things, maybe we can remove this altogether.
2. If this needs to stay, I think a clearer way of doing this would be:
```
@question = @questions[(params[:question_id]).to_i - 1]
@question_text = @question["text"]
@answers = @question["answers"]
```
_(only do this if you can't completely remove these answers controllers though)_